src/main/kotlin/lotto/LottoMachine.kt

```diff
@@ -0,0 +1,34 @@
+package lotto
+
+class LottoMachine(val purchaseAmount: Int, val tickets: MutableList<Lotto> = emptyList<Lotto>().toMutableList()) {
+    private val numberList = (Lotto.Companion.MIN..Lotto.Companion.MAX)
+    private var change = 0
+    private var ticketCount = 0
```

Comment:
As the name "companion object" suggests, we could naturally approach it this way!

```suggestion
    private val numberList = (Lotto.MIN..Lotto.MAX)
    private var change = 0
    private var ticketCount = 0
```

Moreover, if it's a fixed value that all object instances use identically...
how about placing it in the companion object area? 😉

```kotlin
companion object {
    private val LOTTO_NUMBER_RANGE = (Lotto.MIN..Lotto.MAX)
    // ...
}
```